assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -147,53 +147,47 @@ describe( 'core/site site info', () => {
 
 		describe( 'setKeyMetricsSetupCompleted', () => {
 			it( 'sets the keyMetricsSetupCompleted property', () => {
-				registry
-					.dispatch( CORE_SITE )
-					.setKeyMetricsSetupCompleted( true );
+				registry.dispatch( CORE_SITE ).setKeyMetricsSetupCompleted( 1 );
 
 				expect(
 					registry.select( CORE_SITE ).isKeyMetricsSetupCompleted()
 				).toBe( true );
```

Comment:
I realize this is how it was before but we shouldn't be using selectors to make assertions about actions. These should test the effect of the action (state change, request, etc).